diff --git a/node_modules/minecraft-protocol/src/ping.js b/node_modules/minecraft-protocol/src/ping.js
index f8328df..c05961d 100644
--- a/node_modules/minecraft-protocol/src/ping.js
+++ b/node_modules/minecraft-protocol/src/ping.js
@@ -30,7 +30,8 @@ function ping (options) {
   options.closeTimeout = options.closeTimeout || 120 * 1000
   options.noPongTimeout = options.noPongTimeout || 5 * 1000
 
-  const client = new Client(false, version.minecraftVersion)
+  const hideErrors = ([ "0", "false", "no" ].includes(process.env.MINECRAFT_PROTOCOL_HIDE_ERRORS)) ? (false) : (true);
+  const client = new Client(false, version.minecraftVersion, undefined, hideErrors)
   return new Promise((resolve, reject) => {
     client.on('error', function (err) {
       clearTimeout(closeTimer)
